scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -201,6 +198,7 @@ task LoadData {
   meta {
     description: "Load data into BigQuery using the Write Api"
     # Not `volatile: true` since there shouldn't be a need to re-run this if there has already been a successful execution.
+    # TODO Need to revisit the `volatile: true` now that we are querying the database to check for samples there
```

Comment:
see above comments about TODO

Reply:
actually for this one---I'd like to maybe discuss this in mobbing!?!